docs/contrib/CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,128 @@
+# Contributor Covenant Code of Conduct
```

Comment:
The Code of Conduct applies to both OR and ORFS and hence should be referenceable from both.